docs/guides/workspace-and-project-management.rst

```diff
@@ -0,0 +1,39 @@
+================================
+Workspace and Project Management
+================================
+
+Compute Engine (CE) organises workflow runs into ``Workspaces`` and ``projects``.
+As such, in order to submit or interact with a workflows run from the CLI, it is necessary to specify the ``Workspace`` and ``Project`` that contains it [1]_.
+
+There are three mechanisms by which this can be managed. In descernding order of priority these are:
```

Comment:
In relation to my aversion to the word "management":

```suggestion
There are three mechanisms by which this can be specified.
In descending order of priority these are:
```